app/assets/javascripts/react/components/presentational/Circulation/Rooms.js

```diff
@@ -0,0 +1,65 @@
+import React, {Component, PropTypes} from 'react';
+import moment from 'moment';
+
+export const now = moment();
+
+class Rooms extends Component {
+  /**
+   * Automatically fetch rooms available timeout after 10 seconds
+   */
+  setRefreshTimeout() {
+    const refresh = () => {
+      this.props.fetchRoomsAvailableCount("/api/v1/rooms/available/{date}", now);
```

Comment:
You want this string to come from the app state in a property.. `kioskReducer.js` sets the api urls. You should add a new one like `available_rooms: "/api/v1/rooms/available/{date}"` along with the others. Then you should refer to it here as `this.props.api.available_rooms`.